test/wasmBenchmarker/benchmark.py

```diff
@@ -94,31 +96,38 @@ def check_programs(engines, verbose):
     if (verbose): print("Checks done")
 
 
-def get_emcc(verbose):
+def get_emcc(verbose, system_emcc=True):
     emcc_path = None 
+
+    if system_emcc and os.system("emcc --version >/dev/null") == 0:
+        if (verbose): print("Emscripten already installed on system")
```

Comment:
Is this a common style?

Reply:
The line 103? The prints are in same style in this file.